plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -2060,6 +2076,23 @@ public void renameMaterializedView(ConnectorSession session, SchemaTableName sou
         catalog.renameMaterializedView(session, source, target);
     }
 
+    // Information that should be added to every generated Iceberg snapshot, in its summary.
+    // TODO - Consider moving this to the catalog module (on commit).
+    public static Map<String, String> getExtraSummaryMetadata(ConnectorSession session, Optional<String> trinoVersion)
```

Comment:
What's the situation when trino version isn't known?

Reply:
None. Just that I didn't really understand Guice 😅 

Thank you!